### PR TITLE
Call hygen library directly

### PR DESCRIPTION
**Pull request by caioquirino**
This change calls hygen library directly instead of its binary, in order to provide us more ways to override and extend hygen functionality without having to leverage command arguments (and being limited to it)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,9 +21,13 @@
   },
   "author": "Bill Beesley <bill@beesley.dev>",
   "license": "MIT",
+  "optionalDependencies": {
+    "fsevents": "2.1.2"
+  },
   "dependencies": {
     "async-fs-wrapper": "^2.7.0",
     "chalk": "^4.1.2",
+    "enquirer": "^2.3.6",
```
Comment (caioquirino) [inline package.json:30]: This lib is necessary for using the hygen library directly

File: package.json [hunk 2]
```diff
@@ -21,9 +21,13 @@
   },
   "author": "Bill Beesley <bill@beesley.dev>",
   "license": "MIT",
+  "optionalDependencies": {
+    "fsevents": "2.1.2"
```
Comment (caioquirino) [inline package.json:25]: Had to add fsevents as optional library as it started failing the build

File: package.json [hunk 3]
```diff
@@ -48,6 +52,7 @@
     "@semantic-release/github": "^7.2.3",
     "@semantic-release/npm": "^7.1.3",
     "@semantic-release/release-notes-generator": "^9.0.3",
+    "@types/jsdom": "^16.2.14",
```
Comment (caioquirino) [inline package.json:55]: The jsdom library updated and the type was incorrect. Pinned to the one that babel is using

**Review by bbeesley — APPROVED** [review]
lgtm

**Pull request merged**